assets/css/nhsuk.css

```diff
@@ -740,54 +740,64 @@ table {
 table th,
 table td {
   font-family: "FS Me", Arial, sans-serif;
-  font-size: 16px;
+  font-size: 20px;
   font-weight: 400;
-  line-height: 24px;
+  line-height: 32px;
   text-transform: none;
-  border-bottom: 1px solid #d9e1e2;
-  padding: 12px 12px 12px 0;
+  border-bottom: 2px solid #d9e1e2;
+  padding: 16px 16px 16px 0;
   vertical-align: top;
 }
 
 @media (min-width: 641px) {
   table th,
   table td {
-    font-size: 16px;
-    line-height: 24px;
+    font-size: 20px;
```

Comment:
If you're using the same combination as the 20px font size we're already using you can use this sass mixin to set font size and line-height: `@include core-font(20)`

It also add the ability to change the font size at different resolutions.